app/static/openapi.yaml

```diff
@@ -620,8 +624,8 @@ components:
           type: object
           description: A data object
         errors:
-          type: array
-          description: List of errors
+          type: object
+          description: The response errors
```

Comment:
We should show what the error object looks like. We do this for the `data` object in a successful response here https://github.com/OperationCode/resources_api/blob/52a1c4b805b8913baa4af58ac67ddf22ba37afd3/app/static/openapi.yaml#L783-L788

Perhaps create a `Error_data` and list out the properties there? We want to make sure all of the properties are available to view.